polyhost/device/poly_kybd.py

```
@@ -58,8 +58,13 @@ def connect(self):
         """Connect to PolyKybd"""
         if not self.hid:
             self.log.debug("Connecting to PolyKybd for the first time...")
-            self.hid = HidHelper(self.device_settings)
-            self.serial = SerialHelper(self.device_settings)
+            try:
+                self.hid = HidHelper(self.device_settings)
+                self.serial = SerialHelper(self.device_settings)
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)